app/script/util/Environment.js

```diff
@@ -197,7 +204,7 @@ window.z.util = z.util || {};
         return app_version();
       }
 
-      const electron_version = this._electron_version(navigator.userAgent);
+      const electron_version = _check.get_electron_version(navigator.userAgent);
```

Comment:
see above. We are not talking about the electron version but our Wire desktop version